bot/cogs/chess/views/_random_chess.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+import discord
+
+from .._types import Piece, Player
+
+
+class RandomChess(discord.ui.View):
```

Comment:
Would either make all of these return ephemeral responses or have a check to make sure the original author is the one who's clicking the buttons unless you want everyone to be able to click them.